### DISCUSSION TITLE
How much SDRAM is needed to reproduce the results in paper?

### DISCUSSION THREAD

**Discussion by YantianZha** [AndrejOrsula/drl_grasping#106] [Q&A] [answered]
Hello,

Could anyone let me know how much SDRAM might be needed to reproduce the results in [paper](https://arxiv.org/pdf/2208.00818.pdf) (Fig. 5)? The experiment is based on GraspPlanetary-DepthImageWithColor-v0 and GraspPlanetary-OctreeWithColor-v0.

Thank you!

**Comment by AndrejOrsula** [comment, marked as answer]
Hello,

As far as I recall, the required system RAM is around 20 GB for experiments with octree observations and full randomization when using the default hyperparameters. With reduced randomization, it is around 16 GB (if you restrict the number of different 3D models).
In both cases, a minimum of 3-4 GB VRAM is required (simulation assets are freed from VRAM when not used).

It is about the same for image observations with the default hyperparameters, as both variants were tuned to be as comparable as possible.

To reduce system RAM usage, the most straightforward approach would be to reduce the size of the replay buffer.
To reduce VRAM usage, you could try to play with the network architecture or decrease the batch size.

**Reply by YantianZha** [reply]
Okay thank you!
